website/src/parsers/js/transformers/jscodeshift/codeExample.txt

```diff
@@ -1,3 +1,7 @@
+// jscodeshift can take a parser, like "babel", "babylon", "flow", "ts", or "tsx"
+// Read more: https://github.com/facebook/jscodeshift#parser
+module.exports.parser = {{parser}}
```

Comment:
Given that we use ES module syntax, should should rather be

```
export const parser = {{parser}}
```

although I would probably even wrap the placeholder into quotes and at least make syntactically valid JavaScript:

```
export const parser = '{{parser}}';
```

Reply:
thanks for the suggestion! i updated the commit with these changes/